docs/operations/rule-configuration.md

```diff
@@ -22,213 +22,304 @@ title: "Retaining or automatically dropping data"
+--header 'Content-Type: application/json' \
+--data-raw '[{
+  "type": "broadcastForever"
+  }]'
+```
+
+To set one or more retention rules for a specific datasource, send a POST request containing a JSON object for each rule to `/druid/coordinator/v1/rules/{datasourceName}`.
+
+The following example request sets a period drop rule and a period broadcast rule for the `wikipedia` datasource:
+
+```bash
+curl --location --request POST 'http://localhost:8888/druid/coordinator/v1/rules/wikipedia' \
+--header 'Content-Type: application/json' \
+--data-raw '[{
+   "type": "dropByPeriod",
+   "period": "P1M",
+   "includeFuture": true
+   },
+   {
+    "type": "broadcastByPeriod",
+    "period": "P1M",
+    "includeFuture": true
+   }]'
+```
+To retrieve all rules for all datasources, send a GET request to `/druid/coordinator/v1/rules`&mdash;for example:
+
+```bash
+curl --location --request GET 'http://localhost:8888/druid/coordinator/v1/rules'
+```
+
+### Rule structure
+
+The rules API accepts an array of rules as JSON objects. The JSON object you send in the API request for each rule is specific to the rules types outlined below.
+
+> You must pass the entire array of rules, in your desired order, with each API request. Each POST request to the rules API overwrites the existing rules for the specified datasource.
+
+The order of rules is very important. The Coordinator reads rules in the order in which they appear in the rules list. For example, in the following screenshot the Coordinator evaluates data against rule 1, then rule 2, then rule 3:
+
+![retention rules](../assets/retention-rules.png)
+
+The Coordinator cycles through all used segments and matches each segment with the first rule that applies. Each segment can only match a single rule.
+
+In the web console you can use the up and down arrows on the right side of the interface to change the order of the rules.
 
 ## Load rules
 
-Load rules indicate how many replicas of a segment should exist in a server tier. **Please note**: If a Load rule is used to retain only data from a certain interval or period, it must be accompanied by a Drop rule. If a Drop rule is not included, data not within the specified interval or period will be retained by the default rule (loadForever).
+Load rules define how Druid assigns segments to [historical process tiers](./mixed-workloads.md#historical-tiering), and how many replicas of a segment exist in each tier.
+
+If you have a single tier, Druid automatically names the tier `_default` and loads all segments onto it. If you define an additional tier, you must define a load rule to specify which segments to load on that tier. Until you define a load rule, your new tier remains empty.
 
-### Forever Load Rule
+### Forever load rule
 
-Forever load rules are of the form:
+The forever load rule assigns segment data to specified tiers. It is the default rule Druid applies to datasources. Forever load rules have type `loadForever`. 
```

Comment:
"assigns segment data to specified" -> "assigns all datasource segments to specified tiers"

I _think_ that is what "forever" means: the entire time range (all segments).

Reply:
Updated.